leaderboard.md

```diff
@@ -0,0 +1,26 @@
+# Leaderboard usage 
+
+
+## How to use the leaderboard
+The Lightning.ai team @lntg has built us a leaderboard on Discord. This is the single best way you can make sure your submissions actually work before the submission, try to beat the unfinetuned toy submission as a starting point.
```

Comment:
lntg id my username :-) (I realize now how close it is to be a contraction of lightning, but it's actually a contraction of lantiga)

Reply:
you landed in the right place it seems haha